src/App.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Routes, Route, Link } from 'react-router-dom';
+import './App.css';
+import CategoryPage from './components/CategoryPage';
+import BookPage from './components/BookPage';
+
+const App = () => (
+  <div className="App">
+    <nav>
+      <h1> Bookstore CMS</h1>
+      <ul>
+        <li>
+          <Link
+            to="/"
+            className="active"
+          >
+            Books
+          </Link>
+        </li>
+        <li>
+          <Link
+            to="/categories"
+          >
+            CATEGORIES
+          </Link>
```

Comment:
✅ Has 2 <Link>s in the navbar which leads to the booklist and categories locations 👍🏻